src/lists/mainnet/mainnet.ts

```diff
@@ -12999,15 +13035,6 @@ export const MAINNET: TokenData[] = [
     logoURI:
       'https://assets.coingecko.com/coins/images/10792/thumb/goldblock.PNG?1585523631',
   },
-  {
-    chainId: 1,
-    address: '0xbcca60bb61934080951369a648fb03df4f96263c',
-    name: 'Aave USDC',
-    symbol: 'AUSDC',
-    decimals: 6,
-    logoURI:
-      'https://assets.coingecko.com/coins/images/14318/thumb/aUSDC.e260d492.png?1615527797',
-  },
```

Comment:
Only noticed afterwards that this one already existed but left the new entry because cleaner data.